src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/ColorPickerPage.xaml

```diff
@@ -216,13 +216,15 @@
                     HorizontalAlignment="Left"
                     Margin="{StaticResource SmallTopBottomMargin}"
                     RelativePanel.Below="DescriptionPanel">
-                <Image x:Uid="ColorPicker_Image" Source="ms-appx:///Assets/Modules/ColorPicker.png" />
+                <HyperlinkButton x:Uid="ColorPicker_ImageHyperlink">
+                    <Image x:Uid="ColorPicker_Image" Source="ms-appx:///Assets/Modules/ColorPicker.png" />
+                </HyperlinkButton>
             </Border>
             <StackPanel x:Name="LinksPanel"
                         Margin="0,1,0,0"
                         RelativePanel.Below="AboutImage"
                         Orientation="Vertical" >
-                <HyperlinkButton NavigateUri="https://aka.ms/PowerToysOverview_ColorPicker">
+                <HyperlinkButton x:Uid="ColorPicker_ImageHyperlink">
```

Comment:
The Uid here should be ColorPicker_HyperLinkToUserHelp or something.  The naming structure needs to be more agnostic for all of them